apps/home/templates/apps_home/overview_page.html

```diff
@@ -3,7 +3,7 @@
 
 {% block content %}
 {% image page.intro_image max-1920x560 as intro_image %}
-<div class="fullwidth ds-block--fullwidth {% if page.background_color %}ds-block--{{ self.background_color }}{% else %}ds-block--body-bg{% endif %}">
+<div class="fullwidth ds-block--fullwidth ds-block--body-bg">
```

Comment:
I think `ds-block--fullwidth ds-block--body-bg` can also go now as they were added to ensure colour was full width but might need testing

Reply:
Cool, will remove!